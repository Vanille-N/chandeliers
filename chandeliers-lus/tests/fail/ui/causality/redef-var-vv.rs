chandeliers_lus::decl! {
    node foo() returns ();
    var a : int; a : int;
    let a = 1; tel
}

fn main() {}
