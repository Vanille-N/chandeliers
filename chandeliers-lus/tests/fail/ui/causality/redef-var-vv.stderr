error: Variable a is declared twice in node foo
 --> tests/fail/ui/causality/redef-var-vv.rs:3:18
  |
3 |     var a : int; a : int;
  |                  ^
